oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/dsl/SampleFamily.java

```diff
@@ -572,7 +591,31 @@ private static boolean doubleComp(CompType compType, double a, double b) {
         }
     }
 
+    private static class K8sRetagOps {
+
+        private static Sample[] pod2Service(Sample[] ss, String serviceNameLabel, String podNameLabel) {
```

Comment:
Merge this in the implementation of `Pod2Service`, as https://github.com/apache/skywalking/pull/6608/files#r601085683 mentioned.

Reply:
done